src/Footer.tsx

```diff
@@ -17,7 +17,7 @@ function FooterContent() {
         <Trans t={t} i18nKey="ui:rightsDisclaimer">Genshin Optimizer is not affiliated with or endorsed by HoYoverse.</Trans>
       </Typography>
       <Typography variant="caption" sx={{ color: grey[200], textAlign: "right" }} >
-        <Trans t={t} i18nKey="ui:appVersion" values={{ version: packageInfo.version }}>Genshin Optimizer Version: <code>{{ version: packageInfo.version }}</code></Trans>
+        <Trans t={t} i18nKey="ui:appVersion" values={{ version: packageInfo.version }}>Genshin Optimizer Version: <a href={`${process.env.REACT_APP_URL_GITHUB_GO}/releases`} style={{ color: "white" }}>{{ version: packageInfo.version }}</a></Trans>
```

Comment:
Make link open in new tab:  `target="_blank" rel="noopener"`

Make font mono-space: `style={{ color: "white", fontFamily: "monospace" }}`